src/containers/permissions/ObjectPermissionsContainer.js

```diff
@@ -92,17 +95,17 @@ const ObjectPermissionsContainer = ({
     filterByPermissionTypes,
     filterByQuery,
     objectKey,
-    permissionsHash,
+    permissions,
     thisUserId,
-    usersHash,
+    users
```

Comment:
Removing the hashes leads to unnecessary renders and basically makes `useMemo` useless. I don't recommend this change until we have a proper approach to avoid `hashCode()`.